platform/x11/joypad_linux.cpp

```diff
@@ -414,7 +414,9 @@ void JoypadLinux::joypad_vibration_start(int p_id, float p_weak_magnitude, float
 	play.type = EV_FF;
 	play.code = effect.id;
 	play.value = 1;
-	write(joy.fd, (const void *)&play, sizeof(play));
+	if (write(joy.fd, (const void *)&play, sizeof(play)) == -1) {
+		return;
```

Comment:
In theory the last two lines don't look like they should be prevented from happening if this fails, so maybe it should just `print_verbose` a warning instead of a `return`. It probably doesn't matter much though.

Reply:
Done.